defold-rive/src/comp_rive.cpp

```diff
@@ -259,7 +400,12 @@ namespace dmRive
         component->m_DoRender = 0;
         component->m_RenderConstants = 0;
 
-        // TODO: Create the bone hierarchy
+        if (!CreateGameobjects(world, component))
+        {
+            dmLogError("Failed to create game objects for paths in the rive model. Consider increasing collection max instances (collection.max_instances).");
```

Comment:
The wording "Consider" means we're not sure what's going on.
This suggests that we're lacking a better error message earlier in the callstack?